docs/learn/projects.md

```diff
@@ -1,3 +1,9 @@
+---
+keywords: [docs, avalanche, AvalancheGO, avalanche CLI, Core Ethereum, Subnet EVM]
+sidebar_label: Projects
```

Comment:
This page can be deleted, and redirected to `https://docs.avax.network/intro` via the `_redirects` file

Reply:
doesn't seem like the change was made

Reply:
committed the change directly to the branch